compiler-rt/lib/sanitizer_common/sanitizer_allocator_dlsym.h

```diff
@@ -77,6 +75,10 @@ struct DlSymAllocator {
     return Realloc(ptr, count * size);
   }
 
+  static uptr Size(void *ptr) {
```

Comment:
Nit: I'd call this `GetSize()` rather than `Size()` because other methods have verbs in their name (e.g. `Allocate`) but `Size` doesn't contain a verb but `GetSize()`.

It's not really that important so feel free to ignore this suggestion.

Reply:
Updated, thanks!